plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1936,6 +1936,76 @@ public void testCreateNestedPartitionedTable()
         dropTable("test_nested_table_3");
     }
 
+    @Test
+    public void testCreateWithNestedPartitionedColTable()
```

Comment:
thanks for the test. please also add one in `TestIcebergSparkCompatibility` to make sure our nested-field-partitioned writes are correctly readable by  spark

we should also test the reverse.

note that TestIcebergSparkCompatibility is a product test, so to run it you need

```
./mvnw clean install -DskipTests # project needs to be built first
bin/ptl test run --environment singlenode-spark-iceberg -- -t TestIcebergSparkCompatibility

Reply:
what is this `bin/ptl` command? I this something that should be created by that `./mvnw clean install -DskipTests ` command? There is no `bin` direftory created  after running the instal of the project on my side locally.

Reply:
It was moved into the `testing` directory `./testing/bin/ptl`

Reply:
It'd be good to have a query with a `WHERE` clause as well, to verify that the predicate pushdown on the nested column works properly. Pushdown on partition columns is guaranteed by the connector so the engine won't repeat the filter.